src/components/Header.astro

```
@@ -52,7 +52,7 @@ const isCurrent = (href: string) =>
     </nav>
 
     <div class="ml-auto flex items-center gap-2 md:gap-4 relative z-10">
-      <img src="/dpsg_ohne.png" alt="DPSG Wort-Bild-Marke" class="hidden md:block h-9 w-auto object-contain drop-shadow-sm" loading="eager" decoding="async" />
+      <img src="/dpsg_ohne.png" alt="DPSG Wort-Bild-Marke" width="228" height="90" class="hidden md:block h-9 w-auto object-contain drop-shadow-sm" loading="eager" decoding="async" />
```

ESLint

[error] Replace `·src="/dpsg_ohne.png"·alt="DPSG·Wort-Bild-Marke"·width="228"·height="90"·class="hidden·md:block·h-9·w-auto·object-contain·drop-shadow-sm"·loading="eager"·decoding="async"` with `⏎········src="/dpsg_ohne.png"⏎········alt="DPSG·Wort-Bild-Marke"⏎········width="228"⏎········height="90"⏎········class="hidden·md:block·h-9·w-auto·object-contain·drop-shadow-sm"⏎········loading="eager"⏎········decoding="async"⏎·····`

(prettier/prettier)